godtools/App/Features/Dashboard/Presentation/Tools/ToolsViewModel.swift

```diff
@@ -95,24 +97,17 @@ class ToolsViewModel: ObservableObject {
             }
             .store(in: &cancellables)
         
-        getUserFiltersUseCase.getUserFiltersPublisher()
-            .flatMap { userFilters in
+        $appLanguage.eraseToAnyPublisher()
+            .flatMap { appLanguage in
                 
-                return Publishers.CombineLatest(
-                    getToolFilterCategoriesUseCase.getCategoryFilterPublisher(with: userFilters.categoryFilterId),
-                    getToolFilterLanguagesUseCase.getLanguageFilterPublisher(from: userFilters.languageFilterId)
-                )
-                .eraseToAnyPublisher()
+                return self.getUserFiltersUseCase.getUserFiltersPublisher(translatedInAppLanguage: appLanguage)
```

Comment:
Capturing self here in flatMap will cause a retain cycle. Easily resolved by using getUserFiltersUseCase that is in the scope of init.